src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -717,41 +760,23 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
         {
             PaPulseAudio_Lock( pulseaudioHostApi->mainloop );
             /* Zero means success */
-            if( ! pa_stream_connect_record( stream->inputStream,
+            if( pa_stream_connect_record( stream->inputStream,
                                             pulseaudioName,
                                             &stream->inputBufferAttr,
                                             pulseaudioStreamFlags ) )
             {
-                pa_stream_set_started_callback( stream->inputStream,
-                                                PaPulseAudio_StreamStartedCb,
-                                                stream );
-            }
-            else
-            {
+                PA_PULSEAUDIO_SET_LAST_HOST_ERROR( 0,
```

Comment:
Still broken here. `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` must only be used when returning `paUnanticipatedHostError` which is not the case here. Also, passing 0 to `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` makes no sense, that is not a pulseaudio error code. You should be passing the error code from pulse audio.